easybuild/easyconfigs/m/M4/M4-1.4.18.eb

```diff
@@ -16,12 +16,20 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['ab2633921a5cd38e48797bf5521ad259bdc4b979078034a3b790d7fec5493fab']
+
+# fix problems occuring from changes in glibc 2.28
+# https://github.com/coreutils/gnulib/commit/4af4a4a71827c0bc5e0ec67af23edef4f15cee8e
```

Comment:
@robertschade Please move these comments to the patch file itself.
You can just add them to the top above the first line (no escaping like `#` needed).

Keep in mind that this will affect the SHA256 checksum of the patch file.